src/integration/java/io/pinecone/PineconeIndexOperationsClientIntegrationTest.java

```diff
@@ -46,6 +47,18 @@ public void createAndDelete() throws IOException {
         List<String> indexList = pinecone.listIndexes();
         assert !indexList.isEmpty();
 
+        // Configure the index
```

Comment:
This is integration test where I'm configuring podtype and replicas which are further verified by describeIndex call.

Reply:
In tests like this, I like to assert what the state was before applying a change. Just to make sure my test is actually doing what I think. For example, [here](https://github.com/pinecone-io/pinecone-ts-client/blob/main/src/integration/control/configureIndex.test.ts#L89), in the typescript integration tests.

Reply:
Added integration tests with edge cases!